ansible/roles-infra/infra-ec2-template-generate/tasks/locate_availability_zones.yml

```diff
@@ -84,7 +84,7 @@
       when: item.rc != 0
       loop: "{{ r_possible_azs.results }}"
       loop_control:
-        label: "{{ item.cmd }}"
+        label: "{{ item.cmd | default(item) }}"
```

Comment:
I don't believe this will fix the issue, because it will just fail on `item.rc` in the `when`.

Reply:
We really should get away from using `blocks` like this with `when` clauses. The ansible code is much cleaner using `include_tasks` with `when`, because this pattern avoids evaluating tasks like this. It makes the ansible output cleaner and avoids weird when clause evaluation like this.

Reply:
Good point